measure-page.md

```diff
@@ -143,26 +168,22 @@ Use this section to explain factors that might lead users to misinterpret or mis
 
 Even quite complex statistical concepts can and should be translated into plain English for users.
 
-Sometimes, it’s helpful to offer a simplified version of a statistical caveat or test, and refer specialist users to ‘Methodology’ or ‘Download the data’ for more detailed and/or technical information.
+Sometimes, it’s helpful to offer a simplified version of a statistical caveat or test, and refer specialist users to ‘Methodology’ or ‘Download the data’ for more detailed and/or technical information. 
 
 See the [Style guide A to Z](/a-z) for boilerplated plain English explanations for specialist concepts.
 
-Suggested maximum word count: 250
-
 ### What the data measures
 
 Use this section to add context and background to the data. For example:
 
 * what this data tells users (for example, ‘This data measures how likely people in different ethnic groups are to own their own homes’)
 * the source of the data (for example, ‘The figures are drawn from the 2014/15 and 2015/16 English Housing Survey (EHS)’)
-* technical aspects that need further explanation (for example, ‘Some households contain people from different ethnic backgrounds; in these circumstances, we have used the ethnic background of the household reference person to define the ethnic background of the household’)
+* technical aspects that need further explanation (for example, ‘some households contain people from different ethnic backgrounds; in these circumstances, we have used the ethnic background of the household reference person to define the ethnic background of the household’)
 * plain English definitions of terms used in the measure (for example, ‘Young people’ means people aged 10 to 17 years’)
-* what’s included where ‘other’ is used to categorise certain parts of the data (for example, ‘Types of crime under ‘Other offences’ include:...’)
+* what’s included where ‘other’ is used to categorise certain parts of the data (for example, ‘Types of crime under ‘Other offences’ include:...’
```

Comment:
I think we needed the closing bracket here?  Or we also need to delete the opening bracket.